build.gradle.kts

```diff
@@ -57,6 +57,8 @@ sourceSets { named("main") { java.srcDirs("src/kotlin") } }
 dependencies {
   signature(libs.animalsniffer.signature.android)
   implementation(libs.hkdf)
+//  implementation(libs.animalsniffer.signature)
+  implementation("org.bouncycastle:bcprov-jdk15on:1.70")
```

Comment:
I added bouncy castle as I think it is outside of the scope of this project to implement cryptographic primitives, and I couldn't find any mature library for AAED ChaCha20Poly1305 as defined in rfc7539 as per the age spec.

That said, I used the bouncy castle lightweight API and tried to avoid JCE.